s3/src/main/scala/org/apache/pekko/stream/connectors/s3/settings.scala

```diff
@@ -13,25 +13,30 @@
 
 package org.apache.pekko.stream.connectors.s3
 
-import java.nio.file.{ Path, Paths }
-import java.time.{ Duration => JavaDuration }
-import java.util.concurrent.TimeUnit
-import java.util.{ Objects, Optional }
-
-import org.apache.pekko
-import pekko.actor.{ ActorSystem, ClassicActorSystemProvider }
-import pekko.http.scaladsl.model.Uri
-import pekko.stream.connectors.s3.AccessStyle.{ PathAccessStyle, VirtualHostAccessStyle }
 import com.typesafe.config.Config
+import org.apache.pekko
+import org.apache.pekko.stream.connectors.s3.impl.S3Request
```

Comment:
If you import org.apache.pekko, then can omit the org.apache bit in later pekko imports. This is preferred.